.travis.yml

```diff
@@ -30,36 +30,33 @@ env:
         - PYTHON_VERSION=3.6
         - NUMPY_VERSION=stable
         - ASTROPY_VERSION=stable
-        - PYTEST_VERSION=3.1
+        # Pull specviz from github until it is released on PyPi and/or conda
+        - SPECVIZ_GIT='git+https://github.com/spacetelescope/specviz#egg=specviz'
```

Comment:
SpecViz is already available through astroconda, but I do believe you'll have to set the astroconda channel explicitly.